tenseal/binding.cpp

```diff
@@ -86,16 +86,31 @@ PYBIND11_MODULE(_tenseal_cpp, m) {
         .def("save_size", &CKKSVector::save_size)
         .def("add", &CKKSVector::add)
         .def("add_", &CKKSVector::add_inplace)
-        .def("add_plain", &CKKSVector::add_plain)
-        .def("add_plain_", &CKKSVector::add_plain_inplace)
+        .def("add_plain", py::overload_cast<double>(&CKKSVector::add_plain))
+        .def("add_plain",
+             py::overload_cast<vector<double>>(&CKKSVector::add_plain))
```

Comment:
It might even be better with const references, right? I think I will open a separate issue for this to do this on the whole project at a time, as I need this PR ASAP.

Reply:
#91

Reply:
const vector references  will improve the performance for sure, now they're copied on every call.